src/App.js

```diff
@@ -1,22 +1,67 @@
 import logo from './logo.svg';
 import './App.css';
+import React from "react";
+import { THEMES, setTheme } from "./theme.js";
 
 function App() {
+
+  // This is a hook.
+  // const [input, setInput] = useState("");
+
+  // I intend to consolidate these functions and rename it.
+  const handleSend1 = async () => {
+    setTheme("Dark 1");
+  }
+  const handleSend2 = async () => {
+    setTheme("Dark 2");
+  }
+  const handleSend3 = async () => {
+    setTheme("Light 1");
+  }
+  const handleSend4 = async () => {
+    setTheme("Light 2");
+  }
   return (
     <div className="App">
       <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+      <nav class="navbar navbar-expand-lg bg-primary" data-bs-theme="dark">
+  <div class="container-fluid">
+    <a class="navbar-brand" href="#">WeSchedule</a>
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarColor01" aria-controls="navbarColor01" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarColor01">
+      <ul class="navbar-nav me-auto">
```

Comment:
It might make more sense to use tabs and buttons instead of links here. Since we are using React, this must be a single-page app, and Bootstrap tabs can simplify switching content for single-page apps.

Reply:
Anchor tags also require href values to pass the semantic checkers (which you have correctly done in the anchor tags here), but using only # changes the URL to add a # at the end when the tags are clicked. Avoiding this while still using anchor tags would require adding a click handler that calls event.preventDefault() for every such anchor tag (and possibly also ensuring all existing click handlers include e.preventDefault()).
The list structure and the associated classes are also unnecessary if a nav with a div with buttons is used (See the second example at https://getbootstrap.com/docs/5.3/components/navs-tabs/#javascript-behavior).